gel/_internal/_codegen/_models/_pydantic.py

```diff
@@ -1277,7 +1278,11 @@ def get_tuple_name(
         t: reflection.NamedTupleType,
     ) -> str:
         names = [elem.name.capitalize() for elem in t.tuple_elements]
-        digest = base64.b64encode(t.uuid.bytes[:4], altchars=b"__").decode()
+        # Base64ing the name is pretty horrific but it avoids needing
+        # to escape it into Python safely.
```

Comment:
Hm, I don't quite understand this -- the name would be a string, right? why not just `repr()` it?

Reply:
It won't be a valid python identifier

Reply:
It will be something like `tuple<a: std::int64, b: array<tuple<std::str, std::bool>>>` or whatever

Reply:
Sure